client/webpack.config.js

```diff
@@ -46,41 +43,46 @@ module.exports = env => {
                   modules: true,
                   localIdentName: "[local]--[hash:base64:5]",
                   minimize: PROD,
-                  sourceMap: !PROD,
-                },
+                  sourceMap: !PROD
+                }
               },
               {
                 loader: "postcss-loader",
                 options: {
                   ident: "postcss",
-                  plugins: (loader) => [
+                  plugins: loader => [
                     require("postcss-import")(),
                     require("postcss-url")(),
                     require("postcss-cssnext")({}),
                     require("postcss-reporter")()
                   ]
                 }
-              },
+              }
             ]
           })
-        },
-      ],
+        }
+      ]
     },
     stats: {
-      children: false,
+      children: false
     },
-    devtool: DEBUG ? 'source-map' : 'hidden-source-map',
+    devtool: DEBUG ? "source-map" : "hidden-source-map",
     plugins: [
-      new CopyWebpackPlugin([{
-        from: path.resolve(SRC_DIRECTORY, 'resources'),
-        to: path.resolve(PUBLIC_DIRECTORY),
-      }]),
+      new CopyWebpackPlugin([
+        {
+          from: path.resolve(SRC_DIRECTORY, "resources"),
+          to: path.resolve(PUBLIC_DIRECTORY)
+        }
+      ])
     ],
     devServer: {
       contentBase: PUBLIC_DIRECTORY,
       port: 3001,
       compress: true,
       open: true,
+      proxy: {
```

Comment:
This is the important part. Not sure how it will work in a production environment, but we can fix that later -- probably just via nginx config.